docs/cloud/Google.md

```diff
@@ -1,7 +1,13 @@
 ---
 title: 'Google Cloud'
 ---
+
 # AlmaLinux OS Google Cloud Platform
+
+:::warning
+Please note the following images are provided by Google not AlmaLinux OS [SIG/Cloud](/sigs/Cloud.html). Therefore, we are not responsible for any defects that are specific to the images provided by Google.
```

Comment:
How about: `Note that the following images are provided by Google, not the [AlmaLinux OS SIG/Cloud](/sigs/Cloud.html). Therefore ...`

Reply:
I like it.

Reply:
Updated and force-pushed.